content/documentation/pages/6-recipes/2-file-ingest/1-sftp-to-jdbc.md

```diff
@@ -0,0 +1,936 @@
+║                           │           │       guid = 60081                                                                                                                                                     ║
+║                           │           │        pid = 71926                                                                                                                                                     ║
+║                           │           │       port = 60081                                                                                                                                                     ║
+║inboundSftp.task-launcher-0│ deployed  │     stderr = /var/folders/hd/5yqz2v2d3sxd3n879f4sg4gr0000gn/T/spring-cloud-deployer-120915912946760306/inboundSftp-1540820991695/inboundSftp.task-launcher/stderr_0.log║
+║                           │           │     stdout = /var/folders/hd/5yqz2v2d3sxd3n879f4sg4gr0000gn/T/spring-cloud-deployer-120915912946760306/inboundSftp-1540820991695/inboundSftp.task-launcher/stdout_0.log║
+║                           │           │        url = http://192.168.64.1:60081                                                                                                                                 ║
+║                           │           │working.dir = /var/folders/hd/5yqz2v2d3sxd3n879f4sg4gr0000gn/T/spring-cloud-deployer-120915912946760306/inboundSftp-1540820991695/inboundSftp.task-launcher             ║
+╚═══════════════════════════╧═══════════╧════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════╝
+
+```
+
+#### Drop a file into the remote directory
+
+Normally data would be uploaded to an SFTP server.
+We will simulate this by copying a file into the directory specified by `--remote-dir`.
+Sample data can be found in the `data/` directory of the [sample project](https://github.com/spring-cloud/spring-cloud-dataflow-samples/blob/master/dataflow-website/recipes/file-ingest/file-to-jdbc/file-to-jdbc.zip?raw=true).
+
+Copy `data/name-list.csv` into the `/tmp/remote-files` directory which the SFTP source is monitoring.
+When this file is detected, the `sftp` source will download it to the `/tmp/local-files` directory specified by `--local-dir`, and emit a Task Launch Request.
+The Task Launch Request includes the name of the task to launch along with the local file path, given as a command line argument.
+Spring Batch binds each command line argument to a corresponding JobParameter.
+The FileIngestTask job processes the file given by the JobParameter named `localFilePath`.
+The `task-launcher` sink polls for messages using an exponential back-off.
+Since there have not been any recent requests, the task will launch within 30 seconds after the request is published.
```

Comment:
Maybe, we could highlight the fact that the timing can be changed to the desired needs. Perhaps also to the docs as to how to change it.

Reply:
I added a tip after create stream and reference it here.